scss/titles.scss

```diff
@@ -4,8 +4,9 @@ $text-big: 24px;
 
 .title {
   font-size: 24px;
-    padding: 32px 0 32px 50px;
-    position: relative;
+  padding: 32px 0 32px 50px;
+  position: relative;
+  margin: 50px 0 0 50px;
```

Comment:
are you sure you want 0 at the bottom? don't you want to separate it from the techs? I think the top margin can be replaced by a padding in the body.

Reply:
by padding on the body everything including the boxes gets padded, making the boxes length bigger than they should